### PR TITLE
Fix group table order criteria

### PR DESCRIPTION
**Pull request by zetahernandez**
Sorting groups criteria
- Points
- GoalsDifference
- GoalsFor

Before
image

After
image

### CODE REVIEW

File: data/worldcupjson/client.go [hunk 1]
```diff
@@ -128,7 +128,17 @@ func (c *Client) GroupTables() ([]data.GroupTable, error) {
...
+			case table[i].Points != table[j].Points:
+				return table[i].Points > table[j].Points
+			case table[i].GoalsDifferential != table[j].GoalsDifferential:
+				return table[i].GoalsDifferential > table[j].GoalsDifferential
+			case table[i].GoalsFor != table[j].GoalsFor:
+				return table[i].GoalsFor > table[j].GoalsFor
+			default:
+				return table[i].Points > table[j].Points
+			}
+
```
Comment (zetahernandez) [inline data/worldcupjson/client.go]:
```suggestion
```

**Comment by muesli** [comment]
Not sure if this will still matter for this year's world cup, but technically speaking there's also a forth tie breaker, if points, goal difference, and goals-for are all equal: the direct comparison.

The fifth and sixth tie breakers are probably out of scope for such a project: fair play record and draw of lots :laughing:

**Review by cedricblondeau — APPROVED** [review]
This is great, thank you!

**Comment by cedricblondeau** [comment]
> Not sure if this will still matter for this year's world cup, but technically speaking there's also a forth tie breaker, if points, goal difference, and goals-for are all equal: the direct comparison.

This is a great point.

I'll go ahead and merge this PR that will fix the group C standings but a PR to support direct comparison would be very appreciated.

**Pull request merged**